### DISCUSSION TITLE
[Bug]: link in docker/main/install_deps.sh GitHub error 404

### DISCUSSION THREAD

**Discussion by spanner3003** [blakeblackshear/frigate#12245] [Report a Bug] [answered]
### Describe the problem you are having

When building docker images for frigate, like I've been doing all weekend, the latest one errored out.
Checked the github repo and got an error 404 the oldest releases is now [Auto-Build 2022-08-31 12:39](https://github.com/BtbN/FFmpeg-Builds/releases/tag/autobuild-2022-08-31-12-39).

### Steps to reproduce

Run `make build` in frigate directory.

### Version

0.14.0 Beta 3

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
#28 [h8l deps 1/3] RUN --mount=type=bind,source=docker/main/install_deps.sh,target=/deps/install_deps.sh     /deps/install_deps.sh
#28 678.0 + [[ arm64 == \a\m\d\6\4 ]]
#28 678.0 + [[ arm64 == \a\r\m\6\4 ]]
#28 678.0 + mkdir -p /usr/lib/btbn-ffmpeg
#28 678.0 + wget -qO btbn-ffmpeg.tar.xz https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2022-07-31-12-37/ffmpeg-n5.1-2-g915ef932a3-linuxarm64-gpl-5.1.tar.xz
#28 ERROR: process "/bin/sh -c /deps/install_deps.sh" did not complete successfully: exit code: 8
```

### Operating system

Debian

### Install method

Docker CLI

### Network connection

Wired

### Camera make and model

N/A

**Comment by NickM-27** [comment, marked as answer]
This was fixed in https://github.com/blakeblackshear/frigate/pull/12243#pullrequestreview-2151760878

**Reply by spanner3003** [reply]
ok sorry well do :-)
